### PR TITLE
fix(skilavottord): base PR for fixing graphql types and adding access control

### PR DESCRIPTION
**Pull request by darrikonn**
# Fix GraphQL Typings

## What

Fix GraphQL typings by using code generator

## Why

They're previously manually written and out-of-date.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/skilavottord/web/screens/Completed/Completed.tsx [hunk 1]
```diff
@@ -40,7 +41,11 @@ export const skilavottordRecyclingRequestQuery = gql`
   }
 `
 
-const Completed = ({ apolloState }: WithApolloProps) => {
+interface PropTypes {
+  apolloState: any
```
Comment (miamiultras) [inline apps/skilavottord/web/screens/Completed/Completed.tsx:45]: can we avoid any here?

File: apps/skilavottord/web/screens/Confirm/Confirm.tsx [hunk 2]
```diff
@@ -47,26 +52,11 @@ const skilavottordVehicleMutation = gql`
...
 `
 
-export interface VehicleMutation {
-  createSkilavottordVehicle: VehicleMutationData
-}
-
-export interface VehicleOwnerMutation {
-  createSkilavottordVehicleOwner: VehicleOwnerMutationData
+interface PropTypes {
+  apolloState: any
```
Comment (miamiultras) [inline apps/skilavottord/web/screens/Confirm/Confirm.tsx:56]: ditto

**Review by miamiultras — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -18,6 +18,7 @@
     "deploy": "nx deploy",
     "test": "nx test",
     "lint": "nx workspace-lint && nx lint",
+    "tsc": "nx tsc --noEmit",
```
Comment (eirikurn) [inline package.json:21]: What does this do?

**Review by eirikurn — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (darrikonn) [reply package.json]: This was a leftover of debugging. Thought I had removed this, good spot.

Removed.

**Review by barabrian — APPROVED** [review]
The island-ui part LGTM

**Pull request merged**